dev/Repeater/ViewManager.cpp

```diff
@@ -677,7 +697,22 @@ winrt::UIElement ViewManager::GetElementFromElementFactory(int index)
         {
             // Set data context only if no x:Bind was used. ie. No data template component on the root.
             auto elementAsFE = element.try_as<winrt::FrameworkElement>();
```

Comment:
>try_as [](start = 39, length = 6)

try_as doesn't make a lot of sense without a check for null. I'd suggest moving this into an else if block instead of an else block. Or if your sure it is a framework element just us as

Reply:
nit: if element is not a framework element, we will av when we try to set the data context. I think it is safe to assume that this will be a framework element or throw an error if it is not.

---
In reply to: [356264008](https://github.com/microsoft/microsoft-ui-xaml/pull/1740#discussion_r356264008) [](ancestors = 356264008)